### PR TITLE
feat: add model columns and fix controlnet script models  and fix refiner models inference error when select no model

### PR DESCRIPTION
**Pull request by o0OooO**
## Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Comment by NingLu** [comment]
We identified new issues on unchanged lines of code. Navigate to the Amazon CodeGuru Reviewer console to view the recommendations to fix them.

File: aws_extension/sagemaker_ui_tab.py [hunk 1]
```diff
@@ -296,7 +297,7 @@ def _list_models(username, user_token):
         if model['allowed_roles_or_users']:
             allowed = ', '.join(model['allowed_roles_or_users'])
         models.append([model['name'], model['type'], allowed,
-                       'In-Use' if model['status'] == 'Active' else 'Disabled'])
+                       'In-Use' if model['status'] == 'Active' else 'Disabled', datetime.datetime.fromtimestamp(model['created'])])
```
Comment (NingLu) [inline aws_extension/sagemaker_ui_tab.py:300]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The naive datetime objects are treated by many datetime methods as local times, it is preferred to use aware datetimes to represent times in UTC. The recommended way to create an aware datetime object representing a specific timestamp in UTC is by passing `tzinfo` as an argument to the method.

[Learn more](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects)

**Review by elonniu — APPROVED** [review]

File: scripts/main.py [hunk 2]
```diff
@@ -93,54 +100,130 @@ def after_component(self, component, **kwargs):
...
+                result.append(gr.update(choices=sd_models.checkpoint_tiles()))
+            else:
+                result.append(gr.update(choices=load_model_list(pr.username, pr.username)))
+            result.append(sagemaker_ui.load_lora_models(pr.username, pr.username))
+            max_models = shared.opts.data.get("control_net_unit_count", 10)
+            if max_models > 0:
+                controlnet_models = load_controlnet_list(pr.username, pr.username)
+                for i in range(max_models):
+                    result.append(gr.update(choices=controlnet_models))
+            # sync append fresh button
```
Comment (alvindaiyan) [inline scripts/main.py:194]: typo: fresh button -> refresh button

Comment (alvindaiyan) [reply scripts/main.py:194]: do we only disable the refresh button if generate on cloud?

**Review by alvindaiyan — APPROVED** [review]
ok to me, two takeaways:
1. we have a explicit way to allow user to choose none refiner model, the original webui doesn't (I might be wrong)
2. when user choose a on-cloud model, there will refresh refiner models, controlnet models, lora models and potentially embedding models. This could be a performance issue later on.

**Pull request merged**